.github/workflows/azure-static-web-apps-agreeable-river-007f2f303.yml

```
@@ -1,46 +1,50 @@
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_job:
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    outputs:
+      deploy_url: ${{ steps.builddeploy.outputs.static_web_app_url }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)